packages/translations/README.md

```diff
@@ -2,43 +2,41 @@
 
 A hook which loads the translations from an external source and initializes i18next.
 
-## Getting started
-
-`yarn add @bothrs/translations`
+- [x] Compatible with React Native & Web
 
-## Usage
+## Getting started
 
 ```
-import { useTranslations } from '@bothrs/translations'
+yarn add @bothrs/translations
 ```
 
-`useTranslations` return a boolean which indicates if the translations are loaded. This can be used to [hide the splashscreen](https://docs.expo.dev/versions/latest/sdk/splash-screen/).
+## Usage with Airtable
 
-```
-  const translationsLoaded = useInitI18Next({
+`useAirtableTranslations` returns a boolean which indicates if the translations are loaded. This can be used to [hide the splashscreen](https://docs.expo.dev/versions/latest/sdk/splash-screen/).
+
+```tsx
+import { useAirtableTranslations } from '@bothrs/translations'
+
+export function AirtableExample() {
+  const translationsLoaded = useAirtableTranslations({
     expirationTime: 60 * 1000,
-    startupLanguage: 'nl',
-    fallbackLng: 'nl',
     loadPath: api + 'translations',
-    dataFormatter: data => {
-      return data.translations
-    },
+    // Any other i18next options
+    lng: 'nl',
+    fallbackLng: 'nl',
   })
+  return <div>{translationsLoaded ? i.t('Ready') : 'Loading'}</div>
```

Comment:
Where's the variable `i` coming from?

Also van we use fully written variable names to make it clear what the value of the variable is?

Reply:
done